### PR TITLE
Service: Total volatile organic compound

### PR DESCRIPTION
**Pull request by pelikhan**
This pull request adds a new service definition for JACDAC.

### CODE REVIEW

File: services/tvoc.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    const min_TVOC?: u16 ppm @ min_reading
+
+Minimum measurable value
+
+    const max_TVOC?: u16 ppm @ max_reading
+
+Minimum measurable value
+
+    const conditioning_period?: u16 min @ 0x180
```
Comment (mmoskal) [inline services/tvoc.md:23]: how about u16.16 s ? I would rather not add another unit...

File: services/tvoc.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Total volatile organic compound
```
Comment (pelikhan) [inline services/tvoc.md:1]: this should be VOC

File: services/tvoc.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Measures equivalent TVOC levels.
+
+    identifier: 0x12a5b597
+    extends: _sensor
+    camel: TVOC
```
Comment (pelikhan) [inline services/tvoc.md:7]: VOC

File: services/tvoc.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+## Registers
+
+     ro TVOC: u16 ppm { typical_min=0, typical_max=1187 }  @ reading
```
Comment (pelikhan) [inline services/tvoc.md:11]: same as eCO2

File: services/tvoc.md (hunk 1 above)
Comment (pelikhan) [reply services/tvoc.md:23]: u32 s

**Pull request merged**